columnar/src/backend/columnar/columnar.c

```diff
@@ -162,10 +164,34 @@ columnar_init_gucs()
 							&columnar_enable_dml,
 							true,
 							PGC_USERSET,
-							GUC_NO_SHOW_ALL,
+							0,
 							NULL, 
 							NULL, 
 							NULL);
+
+	DefineCustomBoolVariable("columnar.enable_column_cache",
+							gettext_noop("Enables column based caching"),
+							NULL,
+							&columnar_enable_page_cache,
+							false,
+							PGC_USERSET,
+							0,
+							NULL,
+							NULL,
+							NULL);
+
+	DefineCustomIntVariable("columnar.column_cache_size",
```

Comment:
Can this variable be used in same way as i.e. work_mem ? 
So it can be set  kB, M, ... units ?

Reply:
yes, I can make that change.